spec/javascripts/NotificationsSpec.js

```diff
@@ -29,8 +36,65 @@ describe("Notifications", function() {
     expect($('#notifications_none').hasClass("display_block")).toBe(true);
   });
 
+  it("close tips notifications modal", function() { // This test is failing!!
```

Comment:
So checked out your fork's branch, and this test is failing because it's hitting

```
$('.tip_close_notifications').click(function() {
    hideTipNotifications.call(this);
  });
```
and then 
```
$('.tip_notifications').click(function() {
    debugger;
    hideTipNotifications.call(this);
  });
```

Which means `hideTipNotifications()` is being called twice.